Resources/Prototypes/Catalog/Cargo/cargo_fun.yml

```
@@ -207,3 +207,12 @@
   cost: 10000
   category: cargoproduct-category-name-fun
   group: market
+
+- type: cargoProduct
+  id: FunATV
+  icon:
+    sprite: Objects/Vehicles/atv.rsi
+    state: vehicle
+  product: CrateFunATV
+  cost: 3000
+  group: market
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)